packages/shared/src/utils/phone.ts

```diff
@@ -1,14 +1,43 @@
-import { parsePhoneNumberWithError } from 'libphonenumber-js';
+import { parsePhoneNumberWithError, type PhoneNumber } from 'libphonenumber-js';
+
+/**
+ * Parse phone number to E.164 format.
+ * This method will add a leading `+` sign if the phone number does not start with it.
+ */
+export const parseE164Number = (phone: string) => {
+  if (!phone || phone.startsWith('+')) {
+    return phone;
+  }
+
+  return `+${phone}`;
+};
 
 /**
  * Parse phone number to number string.
- * E.g. +1 (650) 253-0000 -> 16502530000
+ * This method will strip the:
+ *  - leading `+` sign of the country code if any,
+ *  - the leading `0` of the phone number if any,
+ *  - non-digit characters if any.
+ *
+ * @params phone - The phone number to parse.
+ * @params throwError - If true, throw an error if the phone number is invalid. Otherwise, return the original phone number.
+ *
+ * @example
+ * +1 (650) 253-0000 -> 16502530000
+ *
+ * @example
+ * +61 0412 345 678 -> 61412345678
  */
-export const parsePhoneNumber = (phone: string) => {
+export const parsePhoneNumber = (phone: string, throwError?: true) => {
```

Comment:
If `throwError` is defaults to `true`, then the console user details page might crash when user inputs an invalid phone number. We may need to change it to `false` by default